docs/src/pages/theme/root-theme.mdx

```diff
@@ -0,0 +1,280 @@
+for them specifically (again, check
+out <a href="https://xstyled.dev/docs/theme-specification/" target="_blank">theme specification</a>).
+**Anchor** does not use every key from the theme specification and actually adds a few of its own.
+
+###### RootTheme, top level
+```jsx hideTitle
+    {
+        borders: {...},
+        breakpoints: {...},
+        colors: {...},
+        fonts: {...},
+        radii: {...},
+
+        // Anchor specific
+        buttons: {...},
+        inputs: {...},
+        skeleton: {...},
+        typography: {...},
+    }
+```
+
+Because the `RootTheme` is a JavaScript object, creating a theme with new values is as simple as
+importing the `RootTheme` and overwriting the existing values.
+
+**Anchor** also ships with a <Link to="/theme/theme-merge">`themeMerge`</Link> utility function to
+help generate a theme, but it's recommended to understand the basics of the `RootTheme` before
+using it.
+
+```jsx
+import { RootTheme } from '@retailmenot/anchor';
+
+export const MyTheme = {
+    ...RootTheme,
+    fonts: {
+        ...RootTheme.fonts
+        base: 'Comic Sans, sans-serif',
+    }
+};
+```
+
+If `MyTheme` was then given to a site's `ThemeProvider`, all **Anchor** components would use
+`Comic Sans` as their font. Why? Because **Anchor**'s styled components are written to use the keys
+that are supplied by the ThemeProvider. Take a look at the code for the `Typography`
+component <a href="https://github.com/RetailMeNot/anchor/blob/master/src/Typography/Typography.component.tsx#L100" target="_blank">here</a>.
+Notice that the value for `fontFamily` is `base`, not the name of the font. This is the essence of
+theming, a layer of abstraction that easily allows sweeping changes to the look of all styled
+components.
+
+This doesn't just apply to **Anchor** components though. Any styled component that is a child of a
+`ThemeProvider` that has access to that theme can also tap into it.
+
+```jsx
+import styled from '@xstyled/styled-components';
+import { ThemeProvider } from '@retailmenot/anchor';
+import { App } from './components/App'; // This path is just an example
+import { MyTheme } from './utils/MyTheme'; // This path is just an example
+
+const Title = styled('h1')`
+    font-family: base;
+    color: primary.light;
+`;
+
+export const Example = () => (
+    <ThemeProvider theme={MyTheme}>
+        <Title>I'll Be in Comic Sans and purplish!</Title>
+        <App />
+    </ThemeProvider>
+);
+```
+
+*NOTE! It's a good idea to extend the `RootTheme` or override its values rather than create a theme
+from scratch. All **Anchor** components are designed to reference established keys from the
+`RootTheme`.*
+
+<br />
+
+## Breakdown
+
+Now that we know how to use `RootTheme` (or a custom theme based on it), here is a breakdown of
+its key/values.
+
+---
+
+#### borders
+The CSS properties, `border`, `border-top`, `border-right`, `border-bottom`, and `border-left`
```

Comment:
This is a confusing sentence to me. I would like it if it followed the format in the sentence `The CSS properties, color, background-color and border-color can all reference the keys in colors.` (the CSS properties `border`, `border-top`, `border-right`, `border-bottom`, and `border-left`, ''can all reference the keys in borders)

Reply:
It's a slight change, but a good one. I'll update the wording in this and the other instances.

Reply:
woot